src/System.Windows.Forms/src/System/Windows/Forms/ComponentModel/COM2Interop/COM2IVsPerPropertyBrowsingHandler.cs

```diff
@@ -104,7 +102,7 @@ private unsafe void OnGetDynamicAttributes(Com2PropertyDescriptor sender, GetAtt
                 if (sender.CanShow)
                 {
                     // should we hide this?
-                    BOOL pfHide = sender.Attributes[typeof(BrowsableAttribute)].Equals(BrowsableAttribute.No) ? BOOL.TRUE : BOOL.FALSE;
+                    BOOL pfHide = sender.Attributes[typeof(BrowsableAttribute)]!.Equals(BrowsableAttribute.No) ? BOOL.TRUE : BOOL.FALSE;
```

Comment:
`sender.Attributes[Type]` is marked as nullable, so we should do something like this:
```suggestion
                    BOOL pfHide = sender.Attributes[typeof(BrowsableAttribute)] is Attribute browsableAttribute
                                && browsableAttribute.Equals(BrowsableAttribute.No) ? BOOL.TRUE : BOOL.FALSE;
```